app/positiondirection.h

```diff
@@ -0,0 +1,58 @@
+#ifndef POSITIONDIRECTION_H
+#define POSITIONDIRECTION_H
+
+#include <QObject>
+#include <QTimer>
+
+#ifdef MOBILE_OS
+#include <QOrientationSensor>
+#include <QCompass>
+#endif
+
+#include "qgsquickpositionkit.h"
+
+class PositionDirection : public QObject
+{
+    Q_OBJECT
+
+    Q_PROPERTY( qreal direction READ direction NOTIFY directionChanged )
+    Q_PROPERTY( QgsQuickPositionKit *positionKit READ positionKit WRITE setPositionKit NOTIFY positionKitChanged )
+    Q_PROPERTY( const qreal MIN_INVALID_DIRECTION READ minInvalidDirection NOTIFY minInvalidDirectionChanged )
```

Comment:
can we have a "bool" property hasDirection() instead of exporting MIN_INVALID_DIRECTION ? It would be much cleaner (the NOTIFY would be upon directionChanged signal)